src/momento/config/transport/transport_strategy.py

```diff
@@ -71,6 +85,21 @@ def with_root_certificates_pem(self, root_certificates_pem_path: Path) -> GrpcCo
     def get_root_certificates_pem(self) -> Optional[bytes]:
         return self._root_certificates_pem
 
+    def get_max_send_message_length(self) -> Optional[int]:
+        return self._max_send_message_length if self._max_send_message_length is not None else None
+
+    def get_max_receive_message_length(self) -> Optional[int]:
+        return self._max_receive_message_length if self._max_receive_message_length is not None else None
```

Comment:
Can just return the attribute here since the attribute is either an int or `None`.

```suggestion
    def get_max_send_message_length(self) -> Optional[int]:
        return self._max_send_message_length

    def get_max_receive_message_length(self) -> Optional[int]:
        return self._max_receive_message_length
```